src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
+
+  @include media($nav-width) {
+    border-bottom: 1px solid $color-gray-light;
+  }
+
+  a {
+    border-bottom: none;
+  }
+
+  em {
+    display: block;
+    font-size: $h5-font-size;
+    font-style: normal;
+    font-weight: $font-bold;
+    margin: 0;
+
+    @include media($nav-width) {
+      display: block;
+      font-size: 2.4rem;
+      line-height: $heading-line-height;
+    }
+  }
+
+  .usa-search {
+    margin-bottom: 1.5rem;
+
+    @include media($nav-width) {
+      float: right;
+      margin-bottom: 0;
+      max-width: 21.5rem;
+    }
+  }
+}
+
+.usa-navbar {
+  border-bottom: 1px solid $color-gray-light;
+  height: 4rem;
+
+  @include media($nav-width) {
+    border-bottom: none;
+    display: inline-block;
+    height: 103px;
+  }
+}
+
+.usa-logo {
+  float: left;
+  line-height: 4rem;
+  margin-left: 1.5rem;
+
+  @include media($nav-width) {
+    line-height: 7.5rem;
+    margin-left: 0;
+    margin-top: 3.3rem;
+  }
+
+  a {
+    color: $color-base;
+    text-decoration: none;
+  }
+
+  br {
+    display: none;
+
+    @include media($nav-width) {
+      display: block;
+    }
+  }
+}
+
+// scss-lint:disable PropertyCount
+.usa-menu-btn {
+  @include button-unstyled;
+  background-color: $color-primary;
+  color: $color-white;
+  display: inline;
+  float: right;
+  font-size: $h6-font-size;
+  line-height: 4rem;
+  height: 4rem;
+  width: 5.8rem;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  @include media($nav-width) {
+    display: none;
+  }
+
+  &:hover {
+    color: $color-white;
+    background-color: $color-primary-darker;
+  }
+
+  &:visited {
+    color: $color-white;
+  }
+}
+// scss-lint:enable PropertyCount
+
+.usa-overlay {
+  @include position(fixed, 0);
+  background: $color-black;
+  opacity: 0;
+  transition: all 0.2s ease-in-out;
+  visibility: hidden;
+  z-index: 9999;
+
+  &.is-visible {
+    opacity: 0.1;
+    visibility: visible;
+  }
+}
+
+@mixin nav-border {
+  border-bottom: 0.7rem solid $color-primary;
+  padding-bottom: 1rem;
+}
+
+.usa-nav-link {
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
The only time you want to use antialiasing is light text on dark background which is what this is here. Sure we can add a comment.